forms-flow-data-layer/src/graphql/service/submission_service.py

```diff
@@ -33,6 +34,18 @@ async def split_search_criteria(
 
         return webapi_search, mongo_search
 
+    @staticmethod
+    def convert_datetimes_to_string(obj):
```

Comment:
@auslin-aot what are your thoughts on using something like [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch) instead of these `if` statements?